### DISCUSSION TITLE
All jobs triggering simultaneously

### DISCUSSION THREAD

**Discussion by SteveScott** [apache/airflow#26284] [Q&A]
Last month or web server went down, then our scheduler went down a week later. Not sure why (that is another discussion), but I restarted them both from the console in daemon mode. But when I restarted the scheduler, all of my jobs re-started simultaneously! This is a problem because if they are not run in the correct order, it corrupts the data. Is there a way to set the scheduler to not catch-up missed jobs? My intent is that if the scheduler goes down, and I restart it, no jobs trigger, allowing me to carefully manually run them. Also it seems if I modify a DAG it will also re-trigger? I don't want any re-triggering to happen unless it is scheduled or manual, it makes management very nervous and distrustful of the system.

**Comment by potiuk** [comment]
You can pause all your DAGs and unpause them individually. But if your DAG is supposed to be running but was not and it gets restarted, then it will attempt to rerun all the missing DAGs. this is  a basic feature of scheduler.

If you want to prevent certain tasks/dags to be run in parallel. you should utilise  all the various options you have for your DAG definition (depends_on_past, pools. max_active_runs, pools, queues and many others). If some tasks in your DAGs that run in parallel corrupt the data, you should define them in the way that they are not run in parallel - so if catching up breaks your data, this is a problem with your DAGs , not Airflow.
